src/coreComponents/linearAlgebra/CMakeLists.txt

```diff
@@ -65,14 +76,18 @@ if( ENABLE_TRILINOS )
           interfaces/trilinos/EpetraUtils.hpp
           interfaces/trilinos/TrilinosPreconditioner.hpp
           interfaces/trilinos/TrilinosSolver.hpp
-          interfaces/trilinos/TrilinosInterface.hpp )
+          interfaces/trilinos/TrilinosInterface.hpp
```

Comment:
As I'm working with spack I noticed that both Trilinos and PETSc have options to build with both SuiteSparse and SuperLU-Dist, currently we're only building PETSc with SuperLU-Dist. Is this something we should change in the future? I don't know anything about what that would enable but it *might* make the interfaces nicer.

Reply:
Not all the options available in SuiteSparse are exposed through the PETSc interface (see discussion [here](https://github.com/GEOSX/thirdPartyLibs/pull/133#issuecomment-704323455)). I believe it is easier to maintain a `ConvertHypreToSuiteSparseMatrix`, `ConvertEpetraToSuiteSparseMatrix`, and `ConvertPetscToSuiteSparseMatrix` and call SuiteSparse directly. The same reasoning applies to SuperLU-Dist.

Reply:
Thanks!

Reply:
Also we're building against SuperLU_Dist 6.3 which had some backward incompatible changes compared to previous versions. Support for it has [just been added](https://github.com/trilinos/Trilinos/pull/8137) in Trilinos master a few days ago and is not in any release yet.